sdks/python/build.gradle

```diff
@@ -182,6 +182,9 @@ def portableWordCountTask(name, streaming) {
         options += ["--environment_cache_millis=10000"]
       if (project.hasProperty("jobEndpoint"))
         options += ["--job_endpoint=${project.property('jobEndpoint')}"]
+      if (project.hasProperty("environmentType")) {
```

Comment:
Lets add environmentConfig as well without it, environmentType will not always be useful.

```
      if (project.hasProperty("environmentConfig")) {
        options += ["--environment_config=${project.property('environmentConfig')}"]
      }
```

Reply:
added